openr/Main.cpp

```diff
@@ -318,7 +319,9 @@ main(int argc, char** argv) {
     netlinkFibServer->setThreadManager(thriftThreadMgr);
     netlinkFibServer->setNumIOWorkerThreads(1);
     netlinkFibServer->setCpp2WorkerThreadName("FibTWorker");
-    netlinkFibServer->setPort(config->getConfig().fib_port);
+    folly::SocketAddress netlinkFibServerAddr("::1", config->getConfig().fib_port);
+    netlinkFibServer->setAddress(netlinkFibServerAddr);
+
```

Comment:
In newer versions this will be able to be removed and config file can set this. I am sure this can be set in your current run_openr.sh and we don't need this in code ...

https://github.com/facebook/openr/blob/main/example_openr.conf#L19